content-src/asrouter/components/RichText/RichText.jsx

```diff
@@ -24,7 +24,8 @@ export function convertLinks(links, sendClick, doNotAutoBlock, openNewWindow = f
       // Setting the value to false will not include the attribute in the anchor
       const url = action ? false : safeURI(links[linkTag].url);
 
-      acc[linkTag] = (<a href={url}
+      acc[linkTag] = (<a href={url} // eslint-disable-line jsx-a11y/anchor-has-content
+        // eslint was getting a false positive caused by the dynamic injection of content.
```

Comment:
You can add a general override for this file here:
https://github.com/mozilla/activity-stream/blob/23821cf7560b709145ffc5b05ba1dc3a73daeade/.eslintrc.jsx-a11y.js#L7-L14

Although perhaps this inline comment might be better as it's for this one place.